run_with_lambda.py

```
@@ -0,0 +1,301 @@
+            
+            if sync_result['success']:
+                new_entries = sync_result.get('new_entries', 0)
+                if new_entries > 0:
+                    self.stats['new_filings'] += new_entries
+                    logger.info(f"Found {new_entries} new filings from bk-feeder")
+                    
+                    # Process new corporate filings
+                    await self._process_new_filings()
+                
+                return sync_result
+            else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)